sale_view_adjust_cft/data/ir.model.access.csv

```diff
@@ -1,3 +1,3 @@
 id,active,name,model_id/id,group_id/id,perm_read,perm_write,perm_unlink,perm_create
 sale_view_adjust_cft_customer_rank1,True,res.partner customerrank,sale_view_adjust_cft.model_res_partner_rank,base.group_user,True,False,False,False
-sale_view_adjust_cft_customer_rank2,True,res.partner customerrank,sale_view_adjust_cft.model_res_partner_rank,calendar_security_adjust.access_all_areas,True,True,True,True
+sale_view_adjust_cft_customer_rank2,True,res.partner customerrank,sale_view_adjust_cft.model_res_partner_rank,base.group_sale_manager,True,True,True,True

```

Comment:
I'm wondering if we should consolidate the changes on security settings (model access and record rules) in `model_security_adjust_cft` for ease of maintenance.  What do you think?